src/ui/components/AccordionItem.tsx

```diff
@@ -120,16 +126,18 @@ const styles = StyleSheet.create({
     paddingHorizontal: getSpacing(6),
     ...(Platform.OS === 'web' ? { cursor: 'pointer' } : {}),
   },
-  bodyContainer: {
-    position: 'absolute',
-    bottom: 0,
-    width: '100%',
-    paddingBottom: getSpacing(6),
-    paddingHorizontal: getSpacing(6),
-    paddingTop: 0,
-  },
 })
 
+const StyledView = styled.View<{ showChildren: boolean }>(({ showChildren }) => ({
+  display: showChildren ? 'flex' : 'none',
+  position: 'absolute',
+  bottom: 0,
+  width: '100%',
+  paddingBottom: getSpacing(6),
+  paddingHorizontal: getSpacing(6),
+  paddingTop: 0,
+}))
+
 const Title = styled(Typo.Title4).attrs(() => getTitleAttrs(2))({
```

Comment:
J'ai raté un épisode, C'est quoi ce `2` ?

Reply:
Je propose de renommer `getTitleAttrs` -> `getHeading(2)`/`getHeadingTitle(2)`/`getHeadingAttrs(2)`/`getHeadingTitleAttrs(2)`  , car on génère un heading 2, le title étant `this`

Reply:
`getHeadingAttrs` me convient, je fais la modif